testConfig.py

```diff
@@ -14,21 +15,19 @@ class ClusterMode(Enum):
 
 
 class TestConfig:
-    def __init__(self, config_path: str):
-        self.mode = ClusterMode.SINGLE
+    kubeconfig_tenant: str = "/root/kubeconfig.tenantcluster"
+    kubeconfig_infra: str = "/root/kubeconfig.infracluster"
+    kubeconfig_single: str = "/root/kubeconfig.nicmodecluster"
+    kubeconfig_cx: str = "/root/kubeconfig.smartniccluster"
+    mode: ClusterMode = ClusterMode.SINGLE
+    client_tenant: K8sClient
+    client_infra: K8sClient
+    full_config: dict
 
+    def __init__(self, config_path: str):
         with open(config_path, "r") as f:
             contents = f.read()
-            self.fullConfig = safe_load(io.StringIO(contents))
-
-        self.kubeconfig_tenant = "/root/kubeconfig.tenantcluster"
-        self.kubeconfig_infra = "/root/kubeconfig.infracluster"
-        self.kubeconfig_single = "/root/kubeconfig.nicmodecluster"
-        self.kubeconfig_cx = "/root/kubeconfig.smartniccluster"
-        self.client_tenant = None
-        self.client_infra = None
-        self.server_node = None
-        self.client_node = None
```

Comment:
Why are we deleting this fields? Are they not needed?

Reply:
Client_node and server_node were not used. 

Tenant and infra are initialized as before, that part of the code [remains unchanged](https://github.com/wizhaoredhat/ocp-traffic-flow-tests/blob/main/testConfig.py#L36)

Only issue could be having them uninitialized instead of `None` . Although I could add some `asserts`, I believe this [guarantees that they will be initialized](https://github.com/wizhaoredhat/ocp-traffic-flow-tests/blob/main/testConfig.py#L52):
```python        
else:
        logger.error("Cannot Find Kubeconfig.")
        sys.exit(-1)
```